modules/tools/migration/src/main/java/org/wso2/mb/migration/DBConnector.java

```diff
@@ -208,9 +196,7 @@ void insertBindings(List<Binding> bindings) throws SQLException {
             }
             preparedStatement.executeBatch();
         } catch (SQLException e) {
-            e.printStackTrace();
-        } finally {
-            conn.close();
+            logger.error("Error while inserting bindings", e);
```

Comment:
We log the exception here without throwing. In `updateBindings` and  `readBindings` we have thrown the exception. Was this intentional?